nifi-nar-bundles/nifi-amqp-bundle/nifi-amqp-processors/src/main/java/org/apache/nifi/amqp/processors/ConsumeAMQP.java

```diff
@@ -194,6 +194,13 @@ protected synchronized AMQPConsumer createAMQPWorker(final ProcessContext contex
 
             return amqpConsumer;
         } catch (final IOException ioe) {
+            try {
+                connection.close();
+                getLogger().warn("Closed connection at port " + connection.getPort());
+            } catch (final IOException ioe_close) {
```

Comment:
Instead of "ioe_close" "ioeClose" would be better?? Also not sure if log message should be at warn or info level.

Reply:
thanks for the input, just changed it. Kept the log message to level WARN as joewitt affirmed.